lib/geoengineer/gps/gps.rb

```diff
@@ -156,6 +156,20 @@ def expanded_hash
 
   def loop_projects_hash(projects_hash, &block)
     projects_hash.each_pair do |project, environments|
+      # If the environments includes _other, pull out its value, and loop over
+      # all other known environments and add it back in, if it doesn't already
+      # exist. This allows _other to be used as a template for cookie cutter
+      # definitions.
+      if environments.key?("_other")
```

Comment:
I'm trying to understand the choice of `_all` versus `_other` in the name.

Given that it iterates over everything _unless_ you explicitly override, perhaps we should just call this concept `_all` to make that clearer? `_other` makes me think this is more complicated than it is, but I might be misunderstanding.

Reply:
I would prefer _all or _default instead of _other

Reply:
`_other` was suggested by @grahamjenson. `_default` may be reasonable, but I believe his thinking was "other" is clearer than "all", because it will be used in environments other than whats specified.

Reply:
`_all` is a lie as soon as you want  to override an environment. `_default` isnt bad.

Reply:
sgtm for `_default`, I'll update and add to the readme

Reply:
Updated and pushed up.